### PR TITLE
chore(#9172): lint translation files

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

#9172

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9172-lint-translations/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9172-lint-translations/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9172-lint-translations/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: api/resources/translations/messages-fr.properties [hunk 1]
```diff
@@ -1126,20 +1126,20 @@ report.immunization_visit.vaccines_received.received_bcg = BCG
...
+report.immunization_visit.vaccines_received.received_hep_b = Hépatite B
 report.immunization_visit.vaccines_received.received_hpv_1 = HPV 1
 report.immunization_visit.vaccines_received.received_hpv_2 = HPV 2
 report.immunization_visit.vaccines_received.received_hpv_3 = HPV 3
-report.immunization_visit.vaccines_received.received_ipv_1 = 
-report.immunization_visit.vaccines_received.received_ipv_2 = 
-report.immunization_visit.vaccines_received.received_ipv_3 = 
+report.immunization_visit.vaccines_received.received_ipv_1 = Polio Désactivé 1
+report.immunization_visit.vaccines_received.received_ipv_2 = Polio Désactivé 2
+report.immunization_visit.vaccines_received.received_ipv_3 = Polio Désactivé 3
```
Comment (garethbowen) [inline api/resources/translations/messages-fr.properties:1142]: Translations provided by @m5r

File: api/resources/translations/messages-sw.properties [hunk 2]
```diff
@@ -1329,7 +1329,7 @@ targets.pregnancy_registrations.title = Mimba mpya
 targets.this_month.subtitle = Mwezi huu
 targets.vaccines_given.title = Chanjo iliyopewa
 task.date = Tarehe ya kukamilisha
-task.days.left = 
+task.days.left = {DAYS, plural, one{Imesalia siku 1} other{Zimesalia siku \#}}
```
Comment (garethbowen) [inline api/resources/translations/messages-sw.properties:1332]: Translations provided by @Benmuiruri

**Comment by garethbowen** [comment]
This change adds a new build step for validating "supported" translation files, which is defined as English, Spanish, French, and Swahili currently. This should also include Nepali once the translations are complete. The idea is that once this is merged PRs will be blocked from merging until they have provided complete and valid translations for each supported language.

I thought about having the "lint translations" being executed at the same time as other linting steps, but this would mean is executes very early and failure would block unit and e2e tests from running. I decided this would be annoying for development where you may not have all the translations yet but you still want to verify that you haven't broken any tests.

This also removes a bunch of POE specific code because POE is no longer being used.

@latin-panda Can I get a review please?

**Review by latin-panda — APPROVED** [review]
So happy to have this!! Thanks!

**Pull request merged**